### PR TITLE
Feat: Hungarian translation

### PR DESCRIPTION
**Pull request by tomzorz**
Hungarian translation.

Magyar fordítás.

### CODE REVIEW

File: src/translations/hu.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "site_url_title": "Fejlesztői Beállítások",
+  "site_url_link_text": "Localhost URL törlése",
+  "site_url_message": "Úgy néz ki egy helyi szervert futtat. Kérjük adja meg a Netlify oldala URL-jét.",
+  "site_url_label": "Adja meg a Netlify oldala URL-jét",
+  "site_url_placeholder": "a Netlify oldala URL-je",
+  "site_url_submit": "URL beállítása",
+  "message_confirm": "Elküldtünk egy megerősítő levelet e-mailben, kérjük kattintson a linkre a levélben a folytatáshoz.",
+  "message_password_mail": "Elküldtünk egy jelszópótló levelet e-mailtben, kérjük kövesse a linket a levélben a jelszava visszaállításához.",
+  "message_email_changed": "Az e-mail címét frissítettük!",
+  "message_verfication_error": "probléma történt a fiókja megerősítése közben. Kérjük próbálja újra, vagy vegye fel a kapcsolatot egy adminisztrátorral.",
```
Comment (cassidoo) [inline src/translations/hu.json:27]: Should this be capitalized?

File: src/index.ejs [hunk 2]
```diff
@@ -309,6 +309,7 @@
 					<option value="en">English</option>
 					<option value="fr">Fran&ccedil;ais</option>
 					<option value="es">Espa&ntilde;ol</option>
+          <option value="hu">Magyar</option>
```
Comment (cassidoo) [inline src/index.ejs:312]: Add a lil more spacing

File: src/translations/hu.json (hunk 1 above)
Comment (tomzorz) [reply src/translations/hu.json]: Yep, fixed

File: src/index.ejs (hunk 2 above)
Comment (tomzorz) [reply src/index.ejs]: Fixed

**Pull request merged**